packages/cozy-stack-client/src/getIllegalCharacter.js

```diff
@@ -0,0 +1,23 @@
+import isString from 'lodash/isString'
+
+/**
+ * Get the list of illegal characters in the file name
+ *
+ * @public
+ * @param {object} file - object representing the file
+ * @returns {string} illegal characters separated by spaces
+ */
+export const getIllegalCharacters = file => {
+  if (!isString(file.name)) throw new Error('file should have a name property ')
```

Comment:
Is that trailing space wanted?
```suggestion
  if (!isString(file.name)) throw new Error('file should have a name property')
```

Reply:
I would choose "consistency by fixing existing code".

This seems to be a typo, I don't find anything in the code that would require that space. @Crash-- are you aware of any Cozy practice that would imply adding a trailing space on error messages?

Reply:
I think that the trailing space was not wanted at the first place. 

Feel free to update it ;)